ts/nni_manager/common/utils.ts

```diff
@@ -232,12 +232,20 @@ function getIPV4Address(): string {
     // since udp is connectionless, this does not send actual packets.
     const socket = dgram.createSocket('udp4');
     socket.connect(1, '192.0.2.0');
-    cachedIpv4Address = socket.address().address;
+    for (let i = 0; i < 10; i++) {  // wait the system to initialize "connection"
+        await yield_();
```

Comment:
a little confused about why we `yield_()` ten times, what happens when initializing "connection"

Reply:
`socket.connect()` returns instantly, but `socket.address()` is not available until OS finish underlying proceeding.
There is no guarantee on how much time it takes, though all experiments shows 1 yield is enough.

Reply:
got it